Multisig/UI/Transaction/TransactionDetailsViewController/TransactionDetailsViewController.swift

```diff
@@ -347,12 +344,20 @@ class TransactionDetailsViewController: LoadableViewController, UITableViewDataS
                 Tracker.trackEvent(.reviewExecutionLedger)
             })
 
+            // needed to fix 'blinking' issue (screen reloads) when
+            // cancelling ledger signing on the device.
+            // Doing this via this variable 'patch' so that we don't rewrite the ledger implementation just yet.
+            var didSign = false
```

Comment:
I would make this as return parameter from the onClose closure instead of this

Reply:
my point in the comment was that I don't want to touch the ledger implementation right now but rather restructure it later. That's why I'm using this variable here now.